test/truefi2/TrueFiPool2.test.ts

```diff
@@ -763,49 +737,31 @@ describe('TrueFiPool2', () => {
 
     it('in order to borrow from pool it has to have liquidity', async () => {
       await setUtilization(90)
-      const loan1 = await createLoan(
-        loanFactory,
-        borrower,
-        tusdPool,
-        (await tusdPool.poolValue()).div(10).add(1),
-        DAY,
-        0,
-      )
 
-      await expect(lender.connect(borrower).fund(loan1.address))
-        .to.be.revertedWith('TrueFiPool: Insufficient liquidity')
+      const amount = (await tusdPool.poolValue()).div(10).add(1)
+      const mockRateAdjuster = await deployMockContract(owner, TrueRateAdjusterJson.abi)
+      await mockRateAdjuster.mock.fixedTermLoanAdjustment.returns(0)
+      await mockRateAdjuster.mock.rate.returns(0)
+      await mockRateAdjuster.mock.borrowLimit.returns(parseEth(100_000_000))
+      await ftlAgency.setRateAdjuster(mockRateAdjuster.address)
```

Comment:
Ah, I see what setRateAdjuster was for.

For tests like this, would it work to pass this mockRateAdjuster in to setupTrueFi2 as an argument?